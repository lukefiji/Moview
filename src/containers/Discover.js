import React, { Component } from 'react';
import { connect } from 'react-redux';

export class Discover extends Component {
  render() {
    return <div>Discover Feed</div>;
  }
}
