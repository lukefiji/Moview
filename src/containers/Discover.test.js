import React from 'react';
import { shallow } from 'enzyme';

import { Discover } from './Discover';

describe('Discover', () => {
  const discover = shallow(<Discover />);

  it('should render properly', () => {
    expect(discover).toMatchSnapshot();
  });
});
