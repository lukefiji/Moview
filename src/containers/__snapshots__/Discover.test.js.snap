// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Discover should render properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Discover />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": "Discover Feed",
    },
    "ref": null,
    "rendered": "Discover Feed",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": "Discover Feed",
      },
      "ref": null,
      "rendered": "Discover Feed",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
